src/hotspot/share/opto/subnode.cpp

```diff
@@ -1621,12 +1621,12 @@ Node *BoolNode::Ideal(PhaseGVN *phase, bool can_reshape) {
   // Change x u< 1 or x u<= 0 to x == 0
```

Comment:
Update comment.

Reply:
What about updating this comment.

Reply:
Right. Sorry I forgot about that one. I updated the comment.